e2etest/l2tests/l2.go

```diff
@@ -502,7 +502,7 @@ var _ = ginkgo.Describe("L2", func() {
 				L2Advs: []metallbv1beta1.L2Advertisement{emptyL2Advertisement},
 			}
 
-			poolCount, err := config.PoolCount(resources.Pools[0])
+			total, _, _, err := config.PoolCount(resources.Pools[0])
```

Comment:
Can we check v4 and v6 numbers too depending on ipFamily?

Reply:
ok